user-script-register-v2/README.md

```diff
@@ -0,0 +1,19 @@
+# User script
+
+This extension demonstrates the [`browser.userScripts.register()`](https://wiki.developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/userScripts/RegisteredUserScript) API. This API enables an extension to register URL-matching scripts at runtime, scripts that run in isolated sandboxes.
```

Comment:
I guess the link here should be to https://wiki.developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/userScripts/register ? Also I don't think the second sentence is helpful here, probably better just to let people read the docs.